### DISCUSSION TITLE
Does NVIDIA Control Panel settings apply to mpv?

### DISCUSSION THREAD

**Discussion by Obegg** [mpv-player/mpv#14220] [Q&A] [answered]
I use NVIDIA GPU and on NVIDIA Control Panel I have many options to adjust:

`Anisotropic filtering` - x16
`Antialiasing - Gamma correction` - On
`Antialiasing - Mode` - Override any application setting
`Antialiasing - Setting` - x8
`Power management mode` - Prefer maximum performance
`Preferred refresh rate` - Highest available
`Shader Cache Size` - Unlimited
`Texture filtering - Negative LOD bias` - Clamp
`Texture filtering - Quality` - High Quality
`Threaded optimization` - On
`Vulkan/OpenGL present method` - Prefer native

Does any of these settings actually improve quality on mpv?
I have tried to test each setting on and off, and I *think* it does make a difference in quality, but I have no way to make sure since this could very be a placebo effect where the quality might not have improved in the first place and it's just me who thinks that.
So I've left with no options other than ask here (the developers), since obviously you might know more than me.

**Comment by Andarwinux** [comment, marked as answer]
These have no effect on mpv, except for vsync and power management mode. If you are using “Optimisations for windowed games” or  HDR, the Vulkan/OpenGL present method is not an option and will be layered on DXGI swapchain anyway.

**Reply by Obegg** [reply]
First of all - thank you for the quick answer.
I was just about the edit the original post to say I just tested `Vertical sync` to Off and it appears it does make a difference.
I would like to ask - other than the three settings you mentioned, does `hwdec=yes` or `hwdec=no` has any change of how NVIDIA Control panel behaves in regards to mpv?

**Reply by Andarwinux** [reply]
I don't know. Maybe NVIDIA will enable some hacks for video players based on DXVA usage, maybe not.

**Comment by Raza-0007** [comment]
The options in Nvidia CP under `Change Resolution: Use Nvidia color settings` and `Adjust video color settings: With Nvidia Settings`, do have an effect on mpv.

Among the settings you mentioned, the ones under `Manage 3D settings: Program settings`, the only one I have tested was `Monitor Technology: G-Sync or Fixed` and that does have an effect. I am not sure about the others. I assumed they work too, so I usually have Nvidia CP set to `Let the 3D application decide`.

By the way I use mpv with `hwdec=auto` which always selects `hwdec=nvdec` for me

~Raza
